tic-tac-toe.js

```diff
@@ -1,7 +1,123 @@
+//if I want to use moon and stars again:
+// function Player(id, sym) {
+//   this.id = id;
+//   this.symbol = sym;
+// }
+
 function TicTacToe() {
+  this.player_one = 'mulder'
+  this.player_two = 'scully'
+  this.turn = this.player_one;
 
+  //if I want to use moon and stars again:
+  // this.player_one = new Player(1, '🌙');
+  // this.player_two = new Player(2, '⭐️');
 }
 
 TicTacToe.prototype = {
-  
+
+  change_turn: function(player, button) {
+
+//if I want to use moon and stars again:
+    // if (button.text() !== '') {return}
+    // button.text(player.symbol);
+
+
+  // this changes players turn
+    if (player === 'mulder') {
+      this.turn = this.player_two;
+    } else {
+      this.turn = this.player_one;
+    }
+  }
+}
+
+
+$(document).on('ready', function() {
+  var game = new TicTacToe();
+
+
+  $('.play').on('click', function(event) {
+    event.preventDefault();
+
+
+    console.log(game.turn)
+    $(this).toggleClass(game.turn)
+    var button = $(this);
+    console.log(button)
+
+    var board = $('#board')
+    console.log(board)
+    console.log(button)
+
+    if (checkIfWon(board, game.turn)) {
+      console.log(game)
```

Comment:
console.log can be super helpful for debugging but make sure to remove all your console.logs from the final product.